test/server/cards/05_LOF/units/ReyWithPalpatinesPower.spec.ts

```diff
@@ -0,0 +1,296 @@
+            });
+
+            it('should do nothing if no Aggression aspect', async function() {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        leader: 'luke-skywalker#faithful-friend',
+                        base: 'dagobah-swamp',
+                        hand: ['mission-briefing'],
+                        deck: ['rey#with-palpatines-power', 'moisture-farmer'],
+                        groundArena: ['wampa'],
+                    }, player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['concord-dawn-interceptors']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.claimInitiative();
+                context.player2.passAction();
+
+                context.player1.clickPrompt('Done');
+                context.player2.clickPrompt('Done');
+
+                expect(context.player1).toBeActivePlayer();
+                expect(context.rey).toBeInZone('hand');
+            });
+
+            it('should do nothing if in hand and a different card is drawn', async function() {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        base: 'kestro-city',
+                        hand: ['mission-briefing', 'rey#with-palpatines-power'],
+                        deck: ['pyke-sentinel', 'moisture-farmer'],
+                        groundArena: ['wampa'],
+                    }, player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['concord-dawn-interceptors']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.missionBriefing);
+                context.player1.clickPrompt('You');
+                expect(context.player2).toBeActivePlayer();
+            });
+
+            it('should be passable', async function() {
+                await contextRef.setupTestAsync({
+                    phase: 'action',
+                    player1: {
+                        base: 'kestro-city',
+                        hand: ['mission-briefing'],
+                        deck: ['rey#with-palpatines-power', 'moisture-farmer'],
+                        groundArena: ['wampa'],
+                    }, player2: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['concord-dawn-interceptors']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.missionBriefing);
+                context.player1.clickPrompt('You');
+
+                expect(context.player1).toHavePrompt('Trigger the ability \'Reveal Rey to deal 2 damage to a unit and 2 damage to a base.\' or pass');
```

Comment:
nit: this is what the `toHavePassAbilityPrompt` is for. It'd be good to use that in case the surrounding prompt text ever changes.